test/Data/String/StripSpec.hs

```diff
@@ -0,0 +1,19 @@
+module Data.String.StripSpec (main, spec) where
```

Comment:
added by `stack new`

Reply:
No idea what template you used but it generated a lot of crap :P I'll remove the garbage and push.

Reply:
No template at all: just `stack new` as per stack 1.7.1